### PR TITLE
fix: adding missing font assets and add style for landing page

### PR DESCRIPTION
**Pull request by NickSaglioni**
- Added asset fonts for BC Sans;
- Added styling for landing pages;

### CODE REVIEW

File: style-sheets/landing-page-components-overrides.scss [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+.landing-desc {
+    margin-top: 2.5rem;
+    color: colors.$gray-70;
+    font-size: 2rem;
+    font-weight: 400;
+    line-height: 2.5rem;
+    margin-bottom: 3.75rem;
+}
+
+.btn-landing {
```
Comment (MCatherine1994) [inline style-sheets/landing-page-components-overrides.scss:46]: Hi Nick, let's rename this, "btn" is the bootstrap button name, let's just call it "button-landing" or something else

File: style-sheets/landing-page-components-overrides.scss [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    width: 18rem;
+    height: 3rem;
+    text-align: inherit;
+    margin-right: 0.625rem !important;
+    font-size: var(--bcgov-body-compact-01-font-size, 0.875rem);
```
Comment (MCatherine1994) [inline style-sheets/landing-page-components-overrides.scss:51]: these "--bcgov-body-compact-01-font-size" looks like some variable from FAM, can we find what it is and if we could use the design token for these?

Comment (OlgaLiber2) [reply style-sheets/landing-page-components-overrides.scss:51]: @MCatherine1994 @NickSaglioni  Token for "body-compact-01" in the design system is $body-compact-01. The token should be inheriting these properties:  
Capture-body-token.

Here is the reference link for the body type in Figma: https://www.figma.com/file/KDXuYanyOsBCM5vpPmNvGf/FSA-Design-System?node-id=11227-270859&t=dNX6YoB5RpIasQjk-4

File: style-sheets/landing-page-components-overrides.scss (hunk 1 above)
Comment (NickSaglioni) [reply style-sheets/landing-page-components-overrides.scss:46]: Thanks, I will change it.

File: design-tokens/type-family.scss [hunk 3]
```diff
@@ -1,53 +1,70 @@
...
+/// @group @carbon/type
+@function font-weight($weight) {
+    @return map-get($font-weights, $weight);
+}
+
+/// Set the `font-weight` property with the value for a given name
+/// @param {String} $weight
+/// @access public
+/// @group @carbon/type
+@mixin font-weight($weight) {
```
[the remark this answers is not in the document]
Comment (NickSaglioni) [reply design-tokens/type-family.scss]: This has been removed as it is not being used right now

**Review by MCatherine1994 — APPROVED** [review]

**Pull request merged**